packages/domain-picker/src/domain-picker/index.tsx

```diff
@@ -94,20 +87,26 @@ const DomainPicker: FunctionComponent< Props > = ( {
 	onMoreOptions,
 	quantity = PAID_DOMAINS_TO_SHOW,
 	currentDomain,
-	domainSearch,
-	onSetDomainSearch,
-	domainCategory,
-	onSetDomainCategory,
-	domainSuggestions,
 	analyticsFlowId,
 	analyticsUiAlgo,
-	domainCategories,
 	domainSuggestionVendor,
+	domainSearch,
+	onSetDomainSearch,
 } ) => {
 	const { __ } = useI18n();
 	const label = __( 'Search for a domain' );
 
 	const [ currentSelection, setCurrentSelection ] = useState( currentDomain );
+	const [ domainCategory, setDomainCategory ] = useState< string | undefined >();
+
+	const [ debouncedDomainSearch ] = useDebounce( domainSearch, 100 );
```

Comment:
Is there a reason we added another debounce function here? We are currently applying debounce to domain API queries in the `useDomainSuggestions` hook. The debounced update of the `domainSearch` value here is making the input to be unresponsive: https://github.com/Automattic/wp-calypso/pull/43045#pullrequestreview-426822928

Reply:
No this is definitely was left here by mistake.

Reply:
Removed in 871f54c0549d9c3b9cab0f9b09d024763af52566